versions/src/main/java/cc/polyfrost/oneconfig/internal/OneConfig.java

```diff
@@ -81,6 +83,8 @@
 //#endif
 public class OneConfig {
 
+    public static final OneConfig INSTANCE = new OneConfig();
+
     public OneConfig() {
```

Comment:
@ApiStatus.Internal ?

Reply:
if it's in oneconfig/internal is it necessary to mark it as such?